src/pages/[spaceId]/[...topic].tsx

```diff
@@ -48,6 +49,24 @@ function getRoomIdAndChatId(topicParams: string[]) {
   return [roomId, chatId] as const
 }
 
+async function getPostsData(roomId: string, chatId: string) {
+  let roomIdAndChatId = [roomId]
+  if (chatId) roomIdAndChatId.push(chatId)
```

Comment:
you use `if` but `chatId` is required in props

Reply:
Fixed typing, chatId should be string | undefined